spec/controllers/comments_controller_spec.rb

```diff
@@ -136,6 +199,10 @@
   end
 
   describe "POST #new" do
+    let(:anon_comment_attributes) do
```

Comment:
So the existing tests are... mistaken, let's say. Much like the `add_comment_reply` action, the `new` actually is actually `GET`, not a `POST`, so you shouldn't need these attributes.